vips/vip-444/bscmainnet.ts

```diff
@@ -0,0 +1,198 @@
+import { BigNumber } from "ethers";
+import { parseUnits } from "ethers/lib/utils";
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { LzChainId, ProposalType } from "src/types";
+import { makeProposal } from "src/utils";
+
+const { POOL_REGISTRY, VTREASURY, NORMAL_TIMELOCK } = NETWORK_ADDRESSES["ethereum"];
+
+export const COMPTROLLER_CORE = "0x687a01ecF6d3907658f7A7c714749fAC32336D1B";
+export const USDT_PRIME_CONVERTER = "0x4f55cb0a24D5542a3478B0E284259A6B850B06BD";
+export const USDC_PRIME_CONVERTER = "0xcEB9503f10B781E30213c0b320bCf3b3cE54216E";
+export const WBTC_PRIME_CONVERTER = "0xDcCDE673Cd8988745dA384A7083B0bd22085dEA0";
+export const WETH_PRIME_CONVERTER = "0xb8fD67f215117FADeF06447Af31590309750529D";
+export const XVS_VAULT_CONVERTER = "0x1FD30e761C3296fE36D9067b1e398FD97B4C0407";
+export const BaseAssets = [
+  "0xdAC17F958D2ee523a2206206994597C13D831ec7", // USDT USDTTokenConverter BaseAsset
+  "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48", // USDC USDCTokenConverter BaseAsset
+  "0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599", // WBTC WBTCTokenConverter BaseAsset
+  "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2", // WETH WETHTokenConverter BaseAsset
+  "0xd3CC9d8f3689B83c91b7B59cAB4946B063EB894A", // XVS XVSTokenConverter BaseAsset
+];
+const CONVERSION_INCENTIVE = parseUnits("3", 14);
+
+type Token = {
+  address: string;
+  decimals: number;
+  symbol: string;
+};
+
+export const token = {
+  address: "0x917ceE801a67f933F2e6b33fC0cD1ED2d5909D88",
+  decimals: 18,
+  symbol: "weETHs",
+};
+
+type Market = {
+  vToken: {
+    address: string;
+    name: string;
+    symbol: string;
+    underlying: Token;
+    decimals: number;
+    exchangeRate: BigNumber;
+    comptroller: string;
+  };
+  riskParameters: {
+    collateralFactor: BigNumber;
+    liquidationThreshold: BigNumber;
+    supplyCap: BigNumber;
+    borrowCap: BigNumber;
+    reserveFactor: BigNumber;
+    protocolSeizeShare: BigNumber;
+  };
+  initialSupply: {
+    amount: BigNumber;
+    vTokenReceiver: string;
+  };
+  interestRateModel: {
+    address: string;
+    base: string;
+    multiplier: string;
+    jump: string;
+    kink: string;
+  };
+};
+
+export const market: Market = {
+  vToken: {
+    address: "0xc42E4bfb996ED35235bda505430cBE404Eb49F77",
+    name: "Venus weETHs (Core)",
+    symbol: "vweETHs_Core",
+    underlying: token,
+    decimals: 8,
+    exchangeRate: parseUnits("1", 28),
+    comptroller: COMPTROLLER_CORE,
+  },
+  riskParameters: {
+    collateralFactor: parseUnits("0.70", 18),
+    liquidationThreshold: parseUnits("0.75", 18),
+    supplyCap: parseUnits("700", 18),
+    borrowCap: parseUnits("0", 18),
+    reserveFactor: parseUnits("0.25", 18),
+    protocolSeizeShare: parseUnits("0.05", 18),
+  },
+  initialSupply: {
+    amount: parseUnits("3", 18),
```

Comment:
Finally, the bootstrap liquidity will be 2

Reply:
https://github.com/VenusProtocol/vips/pull/484/commits/abc5bde91b713fd5fedc9b3c20fde55de85fef70